packages/teleterm/src/mainProcess/protocolHandler.ts

```diff
@@ -0,0 +1,93 @@
+import type {
+  protocol as ElectronProtocol,
+  ProtocolRequest,
+  ProtocolResponse,
+} from 'electron';
+import * as path from 'path';
+import fs from 'fs';
+
+type CallbackType = (response: string | ProtocolResponse) => void;
+
+function _disabledHandler(
```

Comment:
Could you rename it to `disabledHandler`? We don't follow the `_private` convention within. I think it's not even used for unused args here (like `_request` below).

Reply:
Yeah for sure. This was straight from the audit and not really a conscious decision so I appreciate the catch

Reply:
And honestly, since we are only pulling in the index.html file that we are casing ourselves, we shouldn't run into this.

Reply:
Now that I think about it, rather than passing a logger to a bunch of different handler functions, this `_disabledHanlder` was only calling the callback, so I removed it all together and just disabled the schemes with a `forEach`. Much cleaner now